src/main/scala/cromwell/engine/backend/jes/authentication/JesAuthentication.scala

```diff
@@ -0,0 +1,50 @@
+package cromwell.engine.backend.jes.authentication
+
+import cromwell.engine.WorkflowDescriptor
+import cromwell.engine.backend.jes.{JesConfigurationSingleton, JesConfiguration, JesInterface}
+import spray.json.JsObject
+
+import scala.util.{Failure, Success}
+
+/**
+ * This makes JesConnection non visible when mixing a JesAuthentication trait
+ */
+trait JesConnection {
+  val JesConnection: JesInterface
+}
+
+/**
+ * Trait for JesAuthentication
+ */
+trait JesAuthentication { self: JesConfiguration with JesConnection =>
```

Comment:
This is the same pattern which confused me above so I'm guessing it's all the same explanation :)